cnf-certification-test/platform/suite.go

```diff
@@ -173,3 +179,28 @@ func testTainted(env *provider.TestEnvironment) {
 	gomega.Expect(taintedNodes).To(gomega.BeNil())
 	gomega.Expect(errNodes).To(gomega.BeNil())
 }
+
+func testIsRedHatRelease(env *provider.TestEnvironment) {
+	ginkgo.By("should report a proper Red Hat version")
+	failedContainers := []string{}
+	logrus.Infof("looping through %d containers", len(env.Containers))
```

Comment:
logrus.debugf

Reply:
Whoops, this was just a debugging line.  I'll remove in my next commit.